addons/hr_expense/models/hr_expense.py

```diff
@@ -950,8 +950,33 @@ def approve_expense_sheets(self):
                 raise UserError(_("You can only approve your department expenses"))
 
         responsible_id = self.user_id.id or self.env.user.id
+        notification = ''
```

Comment:
You could define default values to have the same type of object returned everytime.
later you can check if your title and type are defined before displaying the notification.
```suggestion
        notification = {
                'type': 'ir.actions.client',
                'tag': 'display_notification',
                'params': {
                    'title': None,
                    'type': None,
                    'sticky': False,  #True/False will display for few seconds if false
                },
            }
```

Reply:
You're totally right. Thanks for your good advice.